R/source_code.R

```diff
@@ -0,0 +1,85 @@
+#' @noRd
+ui_source_code <- function(id) uiOutput(NS(id, "source_code_container"))
+
+#' @noRd
+ui_source_button <- function(id) {
+  shiny::tagList(
+    htmltools::htmlDependency(
+      name = "teal-reporter-busy-disable",
+      version = utils::packageVersion("teal.reporter"),
+      package = "teal.reporter",
+      src = "js",
+      script = "busy-disable.js"
+    ),
+    shiny::actionButton(
+      shiny::NS(id, shiny::NS("source_code", "button")),
```

Comment:
Nice work-around to avoid `verbatim_popup_ui` but not accepting. This is not documented nor commented at all. Because of this `srv_source_code` is confusing where `ui_source_button(session$ns(NULL))` is linked with `teal.widgets::verbatim_popup_srv(id = "source_code")`.

You can consider renaming  `ui_source_button` -> `verbatim_popup_ui` and call it with the same namespace in the renderUI


```suggestion
      shiny::NS(id, "button"),
```

Reply:
Good call, see the new changes in 4bb8ecd42

It "overrides" the `verbatim_popup_ui`